### PR TITLE
[Sql azure] Normalize joomla install sample brochure sql file

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
### Summary of Changes

Normalizes all sql azure joomla **sample_brochure.sql** install file.
- always use `SET IDENTITY_INSERT [#__xxxx] ON/OFF;`
- `INSERT` to `INSERT INTO`
- Quote all table names and db field names
- Always use `'` to quote string values.
- solves several problems in the file
- Code style
### Testing Instructions

Mainly code review. (to many changes need to check code in raw) 
https://github.com/andrepereiradasilva/joomla-cms/blob/patch-23/installation/sql/sqlazure/sample_brochure.sql

If anyone could test an install of **sample brochure** 3.6.x would be great. Don't have sql azure to test,
### Documentation Changes Required

None.

@waader @alikon please test.

### CODE REVIEW

File: installation/sql/sqlazure/sample_brochure.sql [hunk 1]
```diff
@@ -1,184 +1,334 @@
...
+UNION ALL
+SELECT 32, 24, 68, 69, 1, 'com_users.notes.catery.7', 'Uncaterised', '{}'
+UNION ALL
+SELECT 33, 1, 77, 78, 1, 'com_finder', 'com_finder', '{"core.admin":{"7":1},"core.manage":{"6":1}}'
+UNION ALL
+SELECT 34, 27, 19, 20, 3, 'com_content.article.1', 'Home Page Title', '{}'
+UNION ALL
+SELECT 35, 27, 21, 22, 3, 'com_content.article.2', 'About Us', '{}'
+UNION ALL
+SELECT 36, 8, 26, 33, 2, 'com_content.catery.8', 'News', '{}'
```
Comment (alikon) [inline installation/sql/sqlazure/sample_brochure.sql]: what is catery should be category

**Comment by andrepereiradasilva** [comment]
solved

**Comment by waader** [comment]
I have tested this item :white_check_mark: successfully on 3b8733b7881a8991bb7a3a07ea036d30c3ac3993<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11793">issues.joomla.org/joomla-cms/11793</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 3b8733b7881a8991bb7a3a07ea036d30c3ac3993<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11793">issues.joomla.org/joomla-cms/11793</a>.</sub>

**Comment by andrepereiradasilva** [comment]
can we get a RTC here?

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11793">issues.joomla.org/joomla-cms/11793</a>.</sub>

**Pull request merged**